src/shapes/textbox.class.js

```diff
@@ -298,6 +298,15 @@
       return width;
     },
 
+    /**
+     *
+     * @param {string} value
+     * @returns {string[]} array of words
+     */
+    wordSplit: function (value) {
```

Comment:
if this is meant to ease overrides, please let's add a comment here to suggest that to developers